sinks/dashboard/items/standard_display_item.py

```
@@ -216,7 +233,16 @@ def on_data_update(self, stream, payload):
                             t + config.GRAPH_STEP, padding=0)
         # For the numerical readout label
         self.data = float(point)
-        self.numRead.setText(f"{self.data:.{self.decimals}f}")
+        if self.parameters.param('Show Slope of Linear Approx.').value():
+            slope_values: list[str] = []
+            for stream in self.series:
```

Pylint (3.3.7)

[refactor] Redefining argument with the local name 'stream'

(R1704)